notebooks/MAST/HSC/CASJOBS/HCV_API/HCV_API_demo.ipynb

```diff
@@ -0,0 +1,1039 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%load_ext pycodestyle_magic\n",
+    "%flake8_on --ignore E261,E501,W291,W293"
```

Comment:
I think you should delete this cell - it's definitely useful for testing/development, but for the reader of the notebook it's distracting.